### ISSUE TITLE
Feature Suggestion: Auto increment id fields

### ISSUE THREAD

**Issue by zagreusinoz** [vuex-orm/vuex-orm#38]
Have an **option** which will have the ORM automatically increment the id of new records if the id is not supplied.

**Comment by kiaking** [comment]
Sound good! I'll work on to this!

**Comment by nanotronic** [comment]
Not everyone deals with autoincremented IDs. Besides its bad practice.
It would be better to let the dev let plugin an ID-Generator, like an UUID-Generator.

**Comment by kiaking** [comment]
I was planning this to be optional as @zagreusinoz mentioned. I agree not everyone needs increment feature. So, I'm planning to add `increment` method.

```js
class User extends Model {
  static fields () {
    return {
      id: this.increment(),
      name: this.attr('')
    }
  }
}
```

**Comment by zagreusinoz** [comment]
That's great kiaking, exactly what I was thinking so that it's optional.

**Comment by kiaking** [comment]
The feature is implemented by #49. This will enabled in next release!

**Comment by kiaking** [comment]
This is now possible with Vuex ORM 0.16.0!
https://github.com/vuex-orm/vuex-orm/releases/tag/v0.16.0
